setup.cfg

```diff
@@ -0,0 +1,32 @@
+[metadata]
+name = validate
+version = 2.1.6
+author = 'Gina Kim, Arpi Beshlikyan'
+author_email = 'ABeshlikyan@mednet.ucla.edu'
+description = 'Python CLI too to validate different parameters in NF scripts/pipelines'
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/uclahs-cds/public-tool-PipeVal
+project_urls =
+    Bug Tracker = https://github.com/uclahs-cds/public-tool-PipeVal/issues
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: GNU General Public License (GPL)
+    Operating System :: OS Independent
+
+[options]
+package_dir = 
+    = .
+packages = find:
+python_requires = >=3.6
+
+install_requires =
+    mock==4.0.2
+    pytest==6.2.5
```

Comment:
I want to note, in case it comes up later, that the [documentation](https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/) states we should not pin specific versions, however, I disagree with this notion especially in the face of recent supply chain attacks.